MyArrayList/Program.cs

```diff
@@ -204,6 +222,7 @@ public void DeleteN(int num, bool beginning)
 
 
     }
+    
     //Deletes last element
     public void Delete(int index)
```

Comment:
Nameing of method and comment is confusing, What method does id delete element by index, so the name should be DeleteByIndex. The same result must be achieved by using method 
`DeleteBy(index, 1) `
so, there is no point in Delete method at all